[1.20.0] Airgapped updates using CLI (#301)
replicatedhq/kots.io: content/kotsadm/installing/installing-embedded-cluster.md

draft: false
---

> This article refers to installing the Admin Console on an embedded cluster.

Reviewer: Not sure if this is better, but I feel like we should clarify that this creates a cluster with admin console, not installs admin console into a cluster.

```suggestion
This article refers to installing the Admin Console with an embedded cluster.
```

Reviewer: Maybe "along with..."

When running the Admin Console on an existing cluster, refer to the [Installing the Admin Console](/kotsadm/installing/installing-a-kots-app/) documentation.

### Powered by kURL
Replicated KOTS leverages a [deep integration](https://blog.replicated.com/kurl-with-replicated-kots/) with the [Replicated kURL project](https://github.com/replicatedhq/kurl) in order to provide native embedded Kubernetes cluster support.
More documentation on installing with kURL (including [advanced install options](https://kurl.sh/docs/install-with-kurl/advanced-options)) is available at [kurl.sh/docs](https://kurl.sh/docs).

> ### Online Installations

Reviewer: So, the airgap instructions explain installing both admin console and the app, yet this section only covers admin console and gives no instruction for how to actually finish installing your app. Where are those instructions?

cat install.sh | sudo bash -s airgap
```

kURL currently uses `.tar.gz` extension for a `.tar` file, hence the `-o *.tar`.

> Note that the airgapped installer in the above step is not the application installer.  After this command completes, the application can be installed using the application airgap bundle.

Reviewer 2: @codelingobot capture Separate sentences with a linebreak in markdown files.

> Note that the airgapped installer in the above step is not the application installer.  After this command completes, the application can be installed using the application airgap bundle.

Reviewer: This can be clearer. If it's not the application installer, what is it? Does it have a name?

Reviewer: Maybe we need a meta section that clarifies that installing an app to an embedded cluster is a two-step process. First create the cluster with Admin Console included, then start Admin Console and install the application.

Reviewer: Going further, Clarifying that there are two things to download might be important, given this is airgapped, and you need to download them both (plus the license) and *then* run the commands.

```bash
kubectl kots install myapp \
  --airgap-bundle ./myapp-1.0.165.airgap \
  --license-file ./license.yaml \
  --config-values ./config.yaml \
  --namespace default \
  --shared-password password
```

> Applications can also be installed into existing clusters as described in [this doc](/kotsadm/installing/airgap-packages/).

Reviewer: That's not a "doc". I don't like using "here", but it's slightly better.

```suggestion
Applications can also be installed into existing clusters as described [here](/kotsadm/installing/airgap-packages/).
```

Reviewer: Why do we even need to link out to this? This whole page is about embedded clusters. If we want to redirect folks looking for existing cluster installs, shouldn't that be at the top of the doc?
